torch/onnx/symbolic_helper.py

```diff
@@ -1220,21 +1213,21 @@ def __interpolate_helper(
 @_beartype.beartype
 def _unbind_helper(g, self, dim, _outputs):
     if GLOBALS.export_onnx_opset_version < 11:
-        from torch.onnx.symbolic_opset9 import unbind
+        unbind = opset9.unbind
     elif GLOBALS.export_onnx_opset_version <= 12:
-        from torch.onnx.symbolic_opset11 import unbind  # type: ignore[no-redef]
+        unbind = opset11.unbind
     else:
-        from torch.onnx.symbolic_opset13 import unbind  # type: ignore[no-redef]
+        unbind = opset13.unbind
     return unbind(g, self, dim, _outputs)
 
 
 @_beartype.beartype
 def _scatter_helper(g, self, dim, index, src):
     if GLOBALS.export_onnx_opset_version <= 10:
-        from torch.onnx.symbolic_opset9 import scatter
+        scatter = opset9.scatter
     else:
         # for mypy, scatter was imported two lines above
-        from torch.onnx.symbolic_opset11 import scatter  # type: ignore[no-redef]
+        scatter = opset11.scatter
     return scatter(g, self, dim, index, src)
```

Comment:
Looks like the logic here is easy. Instead of assigning, let's call the functions directly in the branches. E.g. opset9.scatter(...) in the first if, opset11.scatter(...) the second